wordpress-mobile/WordPress-Android
# Issue/13329 activity more menu

Parent #13329 

This PR adds the more menu off of the activity log to support Restore and Creating the Downloadable Backup. This logic replaces the previous exposure methodology (dialog). 

<img src="https://user-images.githubusercontent.com/506707/101551218-b3321b00-397e-11eb-922f-ccdfdf297807.png" width="200" alt="Screenshot">

**Merge instructions**
- Make sure ~~#13532~~ and #13545 have been merged
- Remove Not Ready For Merge label
- Merge as normal

**Notes**
- Text & icon color need to be finalized

**To test:**
- Launch the App
- Navigate to Activity Log
- Notice that the action button icon still shows the rewind icon
- Head over to App Settings -> test feature configuration
- Enable the BackupFeatureConfig and restart the app
- Navigate back to Activity Log
- Notice that the action button is now the 3 dot vertical more icon
- Tap on the icon to show the menu options
- Tap the "Download backup" menu item
- Note the backup detail view is launched (The feature itself is not functional, this is just to test that the view shows)
- Go back to the Activity Log
- Tap on the icon to show the menu options
- Tap on the "Restore to this point" menu item
- Note the existing Rewind confirmation dialog is shown


PR submission checklist:
